### ISSUE TITLE
feat: price conversion from lnbits

### ISSUE THREAD

**Issue by dni** [lnbits/lnbits-bitcoin-onchain-and-lightning-payment-gateway#13]
https://github.com/lnbits/woocommerce-payment-gateway/blob/7eb7e3027dbfdf8942599499e65b4417160a9925/includes/utils.php#L6

this could be replaced with a new feature of satspay plugin on lnbits. 
https://github.com/lnbits/satspay/pull/49

**Comment by blackcoffeexbt** [comment]
@dni Does this require the API call to have the "currency" parameter set to a fiat currency e.g. "gbp", "eur", "usd" and the amount to be the fiat value?

**Comment by dni** [comment]
yes, if i  remember correctly,  you create a charge with `currency` and `currency_amount` and do not use the amount
